spec/authentication.yaml

```diff
@@ -179,11 +225,108 @@ components:
             $ref: 'openapi.yml#/components/responses/BadRequest'
           "401":
             $ref: 'openapi.yml#/components/responses/UnauthorizedError'
-          "422":
-            $ref: 'openapi.yml#/components/responses/UnprocessableEntity'
           "500":
             $ref: 'openapi.yml#/components/responses/InternalServerError'
 
         security:
           - basicAuth: []
-            conjurAuth: []
+          - conjurAuth: []
+
+    UpdateAuthenticatorConfig:
```

Comment:
this endpoint is missing the note requested here: https://github.com/cyberark/conjur-openapi-spec/issues/66#issuecomment-758210922